### PR TITLE
fix: sharing of addresses and media

### PR DESCRIPTION
**Pull request by brian-gates**
Bumps teslajsonpy to v3.7.5.

Fixes https://github.com/alandtse/tesla/issues/541.

### CODE REVIEW

**Comment by brian-gates** [comment]
Ok, @alandtse, I think I got it this time! `.devcontainer` was very helpful 🙏🏻 

Let me know if any changes are needed!

File: pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-teslajsonpy = "^3.7.4"
+teslajsonpy = "3.7.5"
```
Comment (alandtse) [inline pyproject.toml:10]: Thanks, actually you can leave the carat in front; it doesn't matter in the end.

**Review by alandtse — CHANGES_REQUESTED** [review]
Thanks. You actually missed an edit you had made in the closed #542, where you bumped manifest.json. Please edit it there too. Once you make that edit, this looks good to go for merge.

File: pyproject.toml (hunk 1 above)
Comment (brian-gates) [reply pyproject.toml]: Done!

**Comment by brian-gates** [comment]
Ok, ready for re-review @alandtse. Thank you for the feedback!

**Comment by alandtse** [comment]
Oh, since you modified the pyproject, your lock will be out of sync. Please run poetry update again to sync it.

**Comment by brian-gates** [comment]
Seems like something isn't right, `poetry update` has been running for over 15 minutes...

```
root ➜ /workspaces/tesla (share-fix-2) $ poetry update
Skipping virtualenv creation, as specified in config file.
Updating dependencies
Resolving dependencies... (937.8s)
```

**Comment by brian-gates** [comment]
image

Ram 📈

**Comment by brian-gates** [comment]
https://github.com/python-poetry/poetry/issues/2094

**Comment by brian-gates** [comment]
This seemed to work!

```
root ➜ /workspaces/tesla (share-fix-2) $ poetry update teslajsonpy@3.7.5
Skipping virtualenv creation, as specified in config file.
Updating dependencies
Resolving dependencies... (2.2s)

Writing lock file

No dependencies to install or update
```

**Comment by brian-gates** [comment]
How's that @alandtse?

**Review by alandtse — APPROVED** [review]

**Pull request merged**